pennylane/__init__.py

```diff
@@ -346,7 +346,7 @@ def run_cnot():
 
         return dev
 
-    raise DeviceError("Device does not exist. Make sure the required plugin is installed.")
+    raise DeviceError(f"Device {name} does not exist. Make sure the required plugin is installed.")
```

Comment:
Looks like this is causing some test failures?

Reply:
Yes, I'll fix this if you don't mind since this might help debugging in the Lightning-Unification repo.